frappe_private_comment/overrides/whitelist/comment.py

```diff
@@ -82,3 +84,40 @@ def get_comment_visibility(name: str):
         return None
 
     return {"custom_visibility": doc.custom_visibility}
+
+
+@frappe.whitelist()
+def get_comment_replies(reference_doctype: str, comment_id: str):
+    """Get comment replies"""
+    return frappe.get_all(
```

Comment:
Where you handle the comment visibility, please handle here if the user should be able to see a given comment or not

Reply:
Added comment visibility